package.json

```diff
@@ -82,13 +82,11 @@
   },
   "dependencies": {
     "bignumber.js": "7.2.1",
-    "ethereumjs-abi": "^0.6.5",
     "ethereumjs-util": "^5.2.0",
```

Comment:
Do we still need ethereumjs-utils?

Reply:
Yes we use some ethereumjs-utils functionalities here and there. Will open an issue to replace this also with `ethers.js`.

Reply:
https://github.com/trustlines-network/clientlib/issues/199